packages/vuetify/src/components/VAutocomplete/VAutocomplete.sass

```diff
@@ -71,6 +71,16 @@
       &:first-child
         margin-inline-start: 0
 
+  &--selecting-index
+    .v-autocomplete__selection
+      opacity: var(--v-medium-emphasis-opacity)
+
+      &--selected
+        opacity: 1
+
+    .v-field__input > input
+      caret-color: transparent !important
```

Comment:
is this possible without the use of `!important`?

Reply:
Yes. I copied this over from VCombobox but it works without `!important` so I have removed it from both VCombobox and VAutocomplete.